lib/winston-syslog.js

```diff
@@ -91,7 +90,24 @@ class Syslog extends Transport {
     this.endOfLine = options.eol;
 
     this.parseProtocol(this.protocol);
-
+    // this options provides a secure socket for sending logs to the syslog server based on
+    // TLSSocket options (https://nodejs.org/api/tls.html#class-tlstlssocket).
+    // Therefore, TLSSocket options are configurable through the code.
+    // Also, to secure the connection, default values of TLS options are set,
+    // which are based on the Protection Profile for Application Software
+    // (https://www.niap-ccevs.org/MMO/PP/-394-/pp_app_v1.2.htm).
+    const ListCiphers = 'ECDHE-ECDSA-AES256-GCM-SHA384\n' +
+    'ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-GCM-SHA384\n' +
+    'ECDHE-ECDSA-AES128-GCM-SHA256\n' +
+    'ECDHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES128-GCM-SHA256\n' +
+    'ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384\n' +
+    'DHE-RSA-AES256-SHA256:ECDHE-ECDSA-AES128-SHA256\n' +
+    'ECDHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA256';
+    this.checkServerIdentity = options.checkServerIdentity || 'undefined'; // check FQDN
```

Comment:
Is string 'undefined' intended, vs. `undefined` itself?

Reply:
I got errors when I ran lint. So I changed from undefined to the string version of 'undefined'. I think it is better to remove the default value or we can set the default value in the const options { } section. What do you think? 

Reply:
Removing the default if it's not a required option seems the most sensible to me.